source/manage-apps/deploy/deployment-models-and-regions.txt

```diff
@@ -32,65 +32,175 @@ Deployment Models
 Global Deployment
 ~~~~~~~~~~~~~~~~~
 
-Globally deployed App Services Apps are hosted in every :ref:`cloud
-region <realm-regions>` that App Services supports. The a Realm SDKs
-automatically execute application requests, such as reading user
-information or calling a function in the cloud region that's closest to
-the user that issued the request.
-
-Write operations to a linked MongoDB cluster are handled within a single
-region that you specify when you :ref:`create your application
-<create-a-realm-app>`. By default, App Services handles writes in
-the region closest to your application's original linked MongoDB
-cluster.
+A globally deployed app is available in multiple regions across the
+world. Any global region can receive and process incoming requests for a
+global app.
+
+All write operations to a linked data source are handled within a single
+region that you specify when you create the app. You can choose any
+global region to handle writes, but should generally choose the region
+closest to your MongoDB cluster.
+
+Global apps are currently deployed to the following regions:
```

Comment:
```suggestion
Global Apps are deployed to all of the following regions:
```

Reply:
also, are global apps only a thing for AWS? if so probably worth calling out

Reply:
Added "_AWS_ regions"